src/Components/HomePageComponent/AuthorisedUser.jsx

```diff
@@ -2,9 +2,8 @@ import React from "react"
 import { navigate, Redirect } from "@reach/router"
 import UserInfo from "./UserInfo"
 import DisplayPosts from "../PostsComponent/DisplayPosts"
-import CreatePosts from "../PostsComponent/CreatePosts"
 
-class Home extends React.Component {
+class AuthorisedUser extends React.Component {
   state = {
```

Comment:
Why not the lift the "posts_arr" state up here to this component?
And pass props down to the child elements for creating new post and displaying posts.